code/notificationbuilder/src/main/res/layout/push_template_zero_bezel_expanded.xml

```diff
@@ -40,6 +40,7 @@
         <TextView
             android:id="@+id/notification_body_expanded"
             style="@style/Body"
+            android:layout_marginBottom="@dimen/zero_bezel_padding_horizontal"
```

Comment:
Does this value exist in `dimens.xml`?

Reply:
Yes 
    <dimen name="zero_bezel_padding_horizontal">16dp</dimen>